examples/sample-pub/publisher.js

```diff
@@ -144,6 +144,43 @@ function startFlowAuto() {
     return;
   }
 
+  if (flow == 'metering') {
+    /* eslint-disable */
+    const timestamp = 1597686771;
+
+    whenReady((subscriptions) => {
+      subscriptions
+        .getEntitlements({
+          metering: {
+            state: {
+              id:
+                'user5901e3f7a7fc5767b6acbbbaa927d36f5901e3f7a7fc5767b6acbbbaa927',
+              standardAttributes: {
+                registered_user: {
+                  timestamp,
```

Comment:
Could we provide some explanation of this structure or an object typedef somewhere?  Or just declare it explicitly so someone can more easily look it up.

Reply:
I added descriptions in `publisher.js` and `entitlements-flow.md`